src/Controllers/CourseController.cs

```diff
@@ -37,5 +37,13 @@ public IActionResult Index(int courseId, ResultsFilter filter)
 
             return View(viewModel);
         }
+        [HttpGet("course/{providerCode:string}/{courseCode:string}/ucas-redirect", Name = "RedirectToUcasCourse")]
+        public RedirectResult RedirectToUcasCourse(string providerCode, string courseCode)
```

Comment:
I don't see anything in this PR that calls this new action. Resharper couldn't find any usages. What am I missing?

We don't want to add dead code. Are you preparing for something else?

Reject?